lib/account/account-service.js

```diff
@@ -1,99 +1,140 @@
 "use strict";
 
 /**
- * What we initially send back to the client so they can start to initiate
- * the complete procedure and save information.
+ * What we initially generate in the registration part of storage
```

Comment:
Please reword.  Remove "what we initially" stuff.  Focus more on describing the idea of the object instead of how we use it.